pkg/operation/seed/seed.go

```diff
@@ -112,21 +112,19 @@ func BootstrapCluster(seed *Seed, k8sGardenClient kubernetes.Client, secrets map
 		return err
 	}
 
-	prometheusVersion, err := imageVector.FindImage("prometheus", k8sGardenClient.Version())
-	if err != nil {
-		return err
-	}
-	configMapReloader, err := imageVector.FindImage("configmap-reloader", k8sGardenClient.Version())
-	if err != nil {
-		return err
-	}
-	pauseContainer, err := imageVector.FindImage("pause-container", k8sGardenClient.Version())
-	if err != nil {
-		return err
-	}
-	gardenerExternalAdmissionController, err := imageVector.FindImage("gardener-external-admission-controller", k8sGardenClient.Version())
-	if err != nil {
-		return err
+	imageNames := []string{
+		common.PrometheusImageName,
+		common.ConfigMapReloaderImageName,
+		common.PauseContainerImageName,
+		common.GardenerExternalAdmissionControllerImageName,
+	}
+	images := make(map[string]string, len(imageNames))
+	for _, imageName := range imageNames {
+		image, err := imageVector.FindImage(imageName, k8sGardenClient.Version(), k8sGardenClient.Version())
```

Comment:
Why not using `GardenVersion()`, similar to `SeedVersion()` and `ShootVersion()`?

Reply:
Which of the two parameters here should be replaced? And also, what should `GardenVersion` be? I mean we're trying to install some stuff on the seed here, so shouldn't at least the `k8sRuntimeVersion` be the version of the seed we're installing on?

Reply:
Ah, ok, wo don't have an `Operation` struct here and the `SeedVersion()` and `ShootVersion()` methods are registered on it. Then I'm fine with keeping it as it is.

> I mean we're trying to install some stuff on the seed here, so shouldn't at least the `k8sRuntimeVersion` be the version of the seed we're installing on?

Definitely, that's a bug. We should for sure use the `k8sSeedClient.Version()`. Please change that! Thanks for pointing this out.